moduleframework/environment_prepare/nspawn_prepare.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Jan Scotka <jscotka@redhat.com>
+#
+
+"""
+module for environment setup and cleanup, to be able to split action for ansible, more steps instead of one complex
+"""
+
+from moduleframework.common import *
+
+selinux_state_file="/var/tmp/mtf_selinux_state"
+setseto = "Permissive"
+
+class EnvNspawn(CommonFunctions):
+
+    def prepare_env(self):
+        self.__prepare()
+
+    def cleanup_env(self):
+        self.__cleanup()
+
+    def __prepare(self):
+        # disable selinux by default if not turned off
+        if not os.environ.get('MTF_SKIP_DISABLING_SELINUX'):
+            print_info("Disabling selinux")
+            # TODO: workaround because systemd nspawn is now working well in F-25
```

Comment:
I have fear that same issue is also in F26 :-) as I discussed this with systemd guys. so yep, I'll rewrite it that it is also in F26